src/app/state/rest/index.ts

```diff
@@ -112,7 +113,17 @@ export const usePanorama = (lat?: number, lon?: number, width?: number, radius?:
 export const usePermitCheckmarks = (bagId: string) => {
   const handleError = useErrorHandler()
   return useApiRequest<{ has_b_and_b_permit: boolean, has_vacation_rental_permit: boolean }>({
-    url: makeGatewayUrl("permits", "get_permit_checkmarks") + `?bag_id=${ bagId }`,
+    url: makeGatewayUrl("permits", "checkmarks") + `?bag_id=${ bagId }`,
+    groupName: "permits",
+    handleError,
+    getHeaders
+  })
+}
+
+export const usePermitDetails = (bagId: string) => {
+  const handleError = useErrorHandler()
+  return useApiRequest<PermitDetailsResponse>({
```

Comment:
Change `PermitDetailsResponse` => `Components.Schemas.DecosPermit[]`. Then PermitDetailResponse.ts can be deleted, I think.